autogluon/task/tabular_prediction/tabular_prediction.py

```diff
@@ -276,6 +276,12 @@ def fit(train_data,
                             problem_types: (list) List of valid problem types for the model. `problem_types=['binary']` will result in the model only being trained if `problem_type` is 'binary'.
                             disable_in_hpo: (bool) If True, the model will only be trained if `hyperparameter_tune=False`.
                         Reference the default hyperparameters for example usage of these options.
+                    _ag_args_fit: Dictionary of model fit customization options related to AutoGluon. These parameters affect stacker fold models, but not stacker models themselves.
```

Comment:
To avoid private variable confusion, consider renaming:
_ag_args -> AG_args
_ag_args_fit -> AG_args_fit

Reply:
also different between `args` and `args_fit` needs to be more clear. `_ag_args` seems too vague a name.

"Dictionary of customization options related to AutoGluon" is too vague a description, in particular "related to AutoGluon" is probably confusing to users.

Reply:
Updated argument names

Updated documentation to clarify differences between them.